tap_jira/streams.py

```diff
@@ -232,23 +232,24 @@ def sync(self):
         timezone = Context.retrieve_timezone()
         start_date = last_updated.astimezone(pytz.timezone(timezone)).strftime("%Y-%m-%d %H:%M")
 
+        stream = Context.get_catalog_entry(self.tap_stream_id)
+        knownFields = stream.schema.properties['fields'].properties
+
         # First, fetch all the custom field names for translation
-        page_num = 0
-        fieldPager = Paginator(Context.client, page_num=page_num, items_key=None)
-        params = {
-            "maxResults": 100,
-        }
         fieldNames = {}
-        for page in fieldPager.pages('issue_fields', "GET", "/rest/api/2/field", params=params):
-            for field in page:
-                id = field['id']
-                name = field['name']
-                fieldNames[id] = name
-            # HACK - there should only be one page, not sure why pager doesn't support this
-            break
+        fields = []
+        for field in Context.client.request('issue_fields', "GET", "/rest/api/2/field"):
+            fields.append(field)
 
-        stream = Context.get_catalog_entry(self.tap_stream_id)
-        knownFields = stream.schema.properties['fields'].properties
+        sortedFields = sorted(fields, key=lambda f: f['custom'], reverse=False)
```

Comment:
Please add some comments explaining what is going on here and what the purpose is of this sorting.